### ISSUE TITLE
Specifying multiple `post_install` hooks is unsupported

### ISSUE THREAD

**Issue by Ewg777** [dtorres/cocoapods-prune-localizations#11]
My Podfile:

``` ruby
source 'https://github.com/CocoaPods/Specs.git'
plugin 'cocoapods-prune-localizations'
inhibit_all_warnings!
use_frameworks!
platform :ios, '8.0'
pod 'Alamofire'

post_install do |installer|
...
end
```

``` ruby
:~ bundle exec pod install --verbose
  Preparing
    - Running pre install hooks
      - cocoapods-prune-localizations from `/usr/local/lib/ruby/gems/2.3.0/gems/cocoapods-prune-localizations-0.3.0/lib/cocoapods_plugin.rb`
[!] Specifying multiple `post_install` hooks is unsupported.
```

**Comment by honghaoz** [comment]
I'm having this issue too. 
`0.2.1` doesn't have this issue. 
`0.3` has.

**Comment by dtorres** [comment]
This error is triggered due to an assertion added in 1.1.0.
Since that version hasn't been released yet it took me a while to track this down.
Fixed in 7d2d8d2
